src/sentry/monitors/clock_dispatch.py

```diff
@@ -85,6 +98,91 @@ def _make_reference_ts(ts: datetime):
     return int(ts.replace(second=0, microsecond=0).timestamp())
 
 
+def _evaluate_tick_decision(tick: datetime):
```

Comment:
How often are ticks sent? Don't we only want this to run once a minute?

Reply:
Yes this is only run once per minute.

It's called just before we call `_dispatch_tick`